pkg/controller/endpointslicemirroring/utils_test.go

```diff
@@ -184,3 +296,121 @@ func newClientset() *fake.Clientset {
 
 	return client
 }
+
+func Test_updateEndpointSliceLabels(t *testing.T) {
+
+	tests := []struct {
+		name string
+		a    map[string]string
+		b    map[string]string
+		want bool
+	}{
+		{
+			name: "both empty",
+			want: false,
+		},
+		{
+			name: "only a empty",
+			b:    map[string]string{"foo": "bar"},
+			want: false,
```

Comment:
Does this mean that any label or annotation that was once set on an Endpoints resource will be mirrored to an EndpointSlice and never removed? Although it's messy, we could potentially compare the new and old Endpoints resource from an update event to try to make sure we catch removals of labels as well.

Reply:
yeah, now I remember we had the same issue with the labels, it seems I keep doing the same mistake :sweat_smile: